src/NuGet.Core/NuGet.Commands/RestoreCommand/RequestFactory/RestoreArgs.cs

```diff
@@ -119,12 +120,18 @@ private List<SourceRepository> GetEffectiveSourcesCore(ISettings settings)
             if (packageSources.Count < 1)
```

Comment:
Maybe add a test that ``--config-file`` works for providing credentials now that I think about it :smile: